src/governance-votedb.cpp

```diff
@@ -34,13 +34,13 @@ bool CGovernanceObjectVoteFile::HasVote(const uint256& nHash) const
     return true;
 }
 
-bool CGovernanceObjectVoteFile::GetVote(const uint256& nHash, CGovernanceVote& vote) const
+bool CGovernanceObjectVoteFile::GetVoteDataStream(const uint256& nHash, CDataStream& ss) const
```

Comment:
Hmm this is not a classical getter. What about `SerializeVoteToStream` as name?

Reply:
Agree, will fix